tests/virtual_machines_test.go

```diff
@@ -32,6 +34,26 @@ var _ = Describe("Virtual Machines", func() {
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	BeforeEach(func() {
```

Comment:
why is BeforeEach not in the context of "Check the client" like AfterEach?

Reply:
It should?

Reply:
I think this would mean BeforeEach will work only once - before "Check the client"

and even if not - BeforeEach & AfterEach should be in the same context, it's confusing otherwise, since we want them to all work before and after each test.

Reply:
done